test/test_integration.rb

```diff
@@ -72,8 +72,12 @@ def start_forked_server(argv)
 
   def stop_forked_server(pid)
     Process.kill(:TERM, pid)
-    sleep 1
-    Process.wait2(pid)
+    sleep 2
+    begin
+      Process.wait2(pid)
+    rescue Errno::ECHILD
+      15
```

Comment:
`15`?

Reply:
@dentarg

Being a windows type, I can't test this locally.

The change above in `stop_forked_server` was done because I did see infrequent (and false) Travis failures from it.  But, it isn't really part of this PR, so I reverted it.

It is wrong (should be `[nil, 15]`), but given the 'green' builds, it shows how infrequent the failures were...  Thanks.